iosHyperskillApp/fastlane/Fastfile

```diff
@@ -239,7 +239,13 @@ platform :ios do
       cocoapods(try_repo_update_on_error: true)
     end
 
-    scan()
+    scan(
+        clean: true,
```

Comment:
`clean` parameter is unnecessary, because `Scanfile` already provides it
https://github.com/hyperskill/mobile-app/blob/develop/iosHyperskillApp/fastlane/Scanfile#L3